modules/client/src/erc20/internal/client/methods.ts

```diff
@@ -72,45 +76,56 @@ export class ClientErc20Methods extends ClientCore
       throw new Error("A web3 provider is needed");
     }
 
-    // TODO: Remove below as the new contracts are ready
-    await delay(1000);
-    yield {
-      key: ProposalCreationSteps.CREATING,
-      txHash:
-        "0x0123456789012345678901234567890123456789012345678901234567890123",
-    };
-    await delay(3000);
-    yield {
-      key: ProposalCreationSteps.DONE,
-      proposalId:
-        "0x1234567890123456789012345678901234567890123456789012345678901234",
-    };
-
-    // TODO: Uncomment as the new contracts are ready
-
-    /*
-    const erc20VotingInstance = ERC20Voting__factory.connect(
-      this._pluginAddress,
-      signer
+    const erc20Contract = ERC20Voting__factory.connect(
+      _params.pluginAddress,
```

Comment:
Can you rename `_params` to `params`, it was changed to avoid TS complaining and not compiling while this was just a mockup, same on the `addressList`